database/index.js

```diff
@@ -0,0 +1,96 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/events');
+
+const db = mongoose.connection;
+
+db.on('error', () => {
+  console.log('Error connecting to the server');
+});
+
+db.once('open', () => {
+  // Start of db open callback
+  console.log('Database connection successful');
+
+  const querySubSchema = mongoose.Schema({
+    eventId: Number,
+    movieObj: {
+      id: Number,
+      profile: {
+        action: Number,
+        animation: Number,
+        comedy: Number,
+        documentary: Number,
+        drama: Number,
+        family: Number,
+        fantasy: Number,
+        international: Number,
+        horror: Number,
+        musical: Number,
+        mystery: Number,
+        romance: Number,
+        sci_fi: Number,
+        thriller: Number,
+        western: Number,
+      },
+      isRec: Boolean,
+    },
+    value: Number,
+  }, { _id: false });
+
+  const userEventsSchema = mongoose.Schema({
+    _id: Number,
+    userId: Number,
+    groupId: Number,
+    events: [querySubSchema],
+  });
+
+  const Event = mongoose.model('event', userEventsSchema);
+
+  const selectAllEvents = ((callback) => {
+    Event.find({}, (err, items) => {
+      if (err) {
+        callback(err, null);
+      } else {
+        callback(null, items);
+      }
+    });
+  });
+
+  const selectSessionEvents = ((session, callback) => {
+    Event.findOne({ _id: session }, (err, item) => {
+      if (err) {
+        callback(err, null);
+      } else {
+        callback(null, item);
+      }
+    });
+  });
+
+  const addEvent = ((session, queryObj) => {
```

Comment:
Is there a reason you're using callbacks instead of promises? This doesn't seem heavily nested or anything, so it's not a problem here, but I can see things getting complicated in the future.